app/javascript/submission_handling/polling.js

```diff
@@ -59,11 +66,22 @@ const deleteData = function (jobId) {
 
 export const renderData = (target, resultCallback, otherRule = null) => {
     pollFetch(target, otherRule).then((result) => {
+        if (checkError(result)) {
+            reportError(result.display_error, result.id);
+        }
+
         resultCallback(result);
         deleteData(target);
     }).
     catch((error) => {
-        // This would be a network error
-        reportError(error);
+        let genericError = { "new_value_formatted": "Error",
+                             "id": target };
+        resultCallback(genericError);
+        reportError('There was a network error, please try again later or call your campus library.', target);
+        // The max wait time was reached. Web Service is probably down.
+        // @todo: create a logging service
+        // eslint-disable-next-line no-console
+        console.error(error);
```

Comment:
```suggestion
```

Reply:
I'd rather keep this until we replace it with a logging service. console.logs (errors, etc) are bad if kept because they are used for debugging, but this is for error reporting. I am reading into your suggestion though, maybe incorrectly. Also, there is a second console.error - did you see that one too?

Reply:
ok cool, I just realized you even added a todo there 👍